site/docs/operations/docker.md

```diff
@@ -359,18 +359,65 @@ To see the default content of the files run the following command:
 docker run hazelcast/hazelcast-jet cat /opt/hazelcast-jet/config/hazelcast-jet.yaml
 ```
 
-## Use Built-in Modules
+## Image Variants
 
-The image comes with several officially supported connectors. You can
-find out the module names on the
-[Sources and Sinks](../api/sources-sinks.md#summary) page.
+Since the Jet version 4.4, Hazelcast Jet docker images come in two
+different variants, these are:
 
-To use one or more modules provide a list of comma separated names in
-`JET_MODULES` environment variable when running the `docker run`
+### Regular image: hazelcast-jet:<version>
```

Comment:
This looks weird, when rendered, `<version>` doesn't show. Just look at this link: https://github.com/ufukyilmaz/hazelcast-jet/blob/update-docker-docs/site/docs/operations/docker.md